libnavigation-core/src/main/java/com/mapbox/navigation/core/preview/RoutesPreview.kt

```diff
@@ -0,0 +1,42 @@
+package com.mapbox.navigation.core.preview
+
+import androidx.annotation.StringDef
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.base.route.NavigationRoute
+import com.mapbox.navigation.core.routealternatives.AlternativeRouteMetadata
+
+@ExperimentalPreviewMapboxNavigationAPI
+data class RoutesPreview(
+    val primaryRoute: NavigationRoute,
+    val alternatives: List<AlternativeRoutePreview>,
+    val routesList: List<NavigationRoute>,
+    val originalRoutesList: List<NavigationRoute>
+)
+
+@ExperimentalPreviewMapboxNavigationAPI
+data class AlternativeRoutePreview(
+    val alternative: NavigationRoute,
+    val metadata: AlternativeRouteMetadata
+)
```

Comment:
This class can be deleted because `AlternativeRouteMetadata` contains the [NavigationRoute](https://github.com/mapbox/mapbox-navigation-android/blob/18e79f5d622ff144afb0cd39bac778369aae538a/libnavigation-core/src/main/java/com/mapbox/navigation/core/routealternatives/AlternativeRouteMetadata.kt#L22) right?

```suggestion
```